### DISCUSSION TITLE
Använda fkui i Nuxt (3)

### DISCUSSION THREAD

**Discussion by gramlin** [Forsakringskassan/designsystem#90] [Q&A] [answered]
Vi på en annan myndighet är flitiga användare av Nuxt. Antar att det bara är att ladda fkui som plugin.

**Comment by dvitamin** [comment, marked as answer]
Jättebra fråga!
Vi har inte haft behovet hitintills, men vi förväntar oss att det kan vara delar som behöver uppdateras för att det ska fungera fullt ut.

Ni får gärna prova och återkomma med feedback. Vi tar gärna emot bidrag med förbättringar kopplat till Nuxt och SSR.

**Reply by gramlin** [reply]
Vi provar oss fram!
